src/main/java/com/scalar/database/transaction/consensuscommit/PrepareMutationComposer.java

```diff
@@ -21,9 +21,11 @@
 import java.util.Optional;
 import javax.annotation.concurrent.NotThreadSafe;
 
+/** Prepare mutations in order to commit to {@link com.scalar.database.api.DistributedStorage}. */
```

Comment:
It is not wrong. I think it is confusing a little.
`PrepareMutation` in this class means the mutation mutates values, metadata and changes the state of record to the `PREPARED` state.

How about this?
```
Compose mutations to prepare records for a transaction
```

Reply:
Thank you! I used your suggestion.